scrapers/IWantClips.yml

```diff
@@ -29,9 +29,9 @@ xPathScrapers:
       Image: //div[@class='clip-thumb-16-9']/a/img/@src
   sceneScraper:
     scene:
-      Title: //span[@class="headline hidden-xs"]/text()
+      Title: //h1[@class="headline hidden-xs"]/text()
       URL:
-        selector: //button[@class='btn btn-default cart addToCart']/@data-id
+        selector: //link[@rel="canonical"]/@href
         postProcess:
           - replace:
               - regex: ^
```

Comment:
Since the link is absolute now we need to remove this postprocessing: tested with this URL, got this result
```
https://iwantclips.com/store/82932/Goddess-Blonde-Kitty/2945599/Had-a-Bad-Day-Watch-This
https://iwantclips.com/store/item/https://iwantclips.com/store/82932/Goddess-Blonde-Kitty/2945599/Had-a-Bad-Day-Watch-This
```